public/i18n/en-us.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
What's the difference between these files and the language *.json* files in the root of */i18n*? Why don't the translations in those files exist here?